sentry-uitest/sentry-uitest-android-benchmark/build.gradle.kts

```diff
@@ -0,0 +1,132 @@
+import io.gitlab.arturbosch.detekt.Detekt
+import io.gitlab.arturbosch.detekt.extensions.DetektExtension
+import net.ltgt.gradle.errorprone.errorprone
+
+plugins {
+    id("com.android.application")
+    kotlin("android")
+    id(Config.QualityPlugins.errorProne)
+    id(Config.QualityPlugins.gradleVersions)
+    id(Config.QualityPlugins.detektPlugin)
+}
+
+configure<JavaPluginExtension> {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
```

Comment:
Do we need that? this is the default version anyway.

Reply:
Maybe not, but I've seen it in all the modules, so I added it here, too, at least to be consistent with the others.
Should i remove it?

Reply:
They are only on Java projects, not on the Android ones.

Reply:
Didn't realize it 😓